action/__main__.py

```diff
@@ -33,6 +33,7 @@
             database=os.environ["DB_NAME"],
             user=os.environ["DB_USER"],
             password=os.environ["DB_PASSWORD"],
+            exclude_tables=os.environ["EXCLUDE_TABLES"].split(","),
```

Comment:
Presumably EXCLUDE_TABLES is optional? In that case, this will raise an error if it's not provided. You have to do `os.getenv()` and split only if it doesn't return None

Reply:
makes sense 👍